Stateless/project.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "2.5.0-*",
+  "dependencies": {
+    "NETStandard.Library": "1.6.0"
+  },
+  "packOptions": {
+    "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+    "projectUrl": "https://github.com/nblumhardt/stateless",
+    "iconUrl": "https://storage.googleapis.com/google-code-archive/v2/code.google.com/stateless/logo.png",
+    "requireLicenseAcceptance": false
+  }, 
+  "authors": [ "Nicholas Blumhardt and Contributors" ],   
+  "description": "Create state machines and lightweight state machine-based workflows directly in .NET code",
+  "language": "en-US",
+  "title": "Stateless",
+  "frameworks": {
+    "netstandard1.6": {
```

Comment:
Looks like we've dropped a lot of platform targets - using project.json it should be possible to keep NET40+ etc.